.github/workflows/ci.yml

```diff
@@ -18,13 +18,21 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: 📚 Install necessary packages
+      shell: bash
+      run:  |
+            if [ "$RUNNER_OS" == "Linux" ]; then
```

Comment:
Instead of using bash's if, please use the GH Actions way of doing it with `if: matrix.os == 'ubuntu-latest'`